src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -275,22 +269,12 @@ public void configureCustomCA(CDAConfiguration configuration) throws InvalidConf
         URI privateKeyUri = configuration.getPrivateKeyUri().get();
         URI certificateUri = configuration.getCertificateUri().get();
 
-        RetryUtils.RetryConfig retryConfig = RetryUtils.RetryConfig.builder()
```

Comment:
Not sure if removing this was right. In my ming if the certificate is written in the FS and it is wrong then no matter how many times you retry it will still be wrong, so I remove the retry. Please educate me if that is a wrong assumption.

Reply:
Yes, this is wrong. It's not here to protect against the key being wrong, it's here because CDA can come up before the SecurityService itself comes up.

FWIW, the retry policy is still wrong. Waiting 200ms is not even remotely sufficient as a backoff. But we can fix that later.

Reply:
Well now we are not using the SecurityService... so I guess we should not need to re add it.